joplin/base/migrations/0030_auto_20180924_1554.py

```diff
@@ -1314,9 +1314,9 @@ def load_service(data):
     for k in ['meta_description_ar', 'meta_description_en', 'meta_description_es', 'meta_description_vi', 'meta_tags', 'meta_title_ar', 'meta_title_en', 'meta_title_es', 'meta_title_vi']:
         data.pop(k, None)
 
-    print(f'-  Loading service steps...\r', end='')
-    service_steps = data.pop('service_steps')
-    print('✅')
+    # print(f'-  Loading service steps...\r', end='')
```

Comment:
Couple things here:
* Instead of commenting let's just remove these lines completely
* Do we have any ideas on how to replicate this functionality with the new steps model?